deployment/k8s/charts/sharedResources/snapshotVolume/templates/pvc.yml

```diff
@@ -1,6 +1,3 @@
-# Only create PVC, if PVC with same name does not already exist
-{{- $pvc := lookup "v1" "PersistentVolumeClaim" .Release.Namespace .Values.volume.pvcClaimName -}}
-{{- if not $pvc }}
```

Comment:
this went away because now we are creating the volume as its own sub-chart?

Reply:
I don't think any component chart should have a sub-chart dependency on a shared resource. I think I have found this structure to be a bit hack-ish and don't want to continue it. We were trying to have this PVC be something that could be shared between components as well as individually created by a component. I've taken the stance now to have this just as a shared PVC that the aggregate chart will create. If we find this is problematic for deploying individual charts I think there are alternatives but this feels cleaner to me